content/news/buzz/december-2023-status-update/contents.lr

```diff
@@ -0,0 +1,60 @@
+title: December 2023 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+December has been a short month due to end-of-year holidays, but BeeWare has still made some important progress.
+
+What we've done
+----------------
+
+* `PEP730 <https://peps.python.org/pep-0730/>`__, formalizing the introduction of iOS as a Tier 3 platform in CPython, has been `ratified by the CPython Steering Council <https://github.com/python/steering-council/issues/218#issuecomment-1839333665>`__. 
+
+* The `first patch to CPython that explicitly mentions iOS <https://github.com/python/cpython/pull/112738>`__ has been merged. This is a relatively minor change that is required for building cross-platform binary modules; we'll start submitting more substantial patches in the new year.
+
+* `PEP738 <https://peps.python.org/pep-0738/>`__, formalizing the introduction of Android as a Tier 3 platform in CPython, has been drafted, and is `currently under discussion <https://discuss.python.org/t/pep-738-adding-android-as-a-supported-platform/40975>`__.
+
+* `Chaquopy 15 was released <https://chaquo.com/chaquopy/chaquopy-version-15-0-1/>`__, adding support for Python 3.12 on Android. This release also improves compatibility with Android Studio's transition away from Groovy, towards Kotlin for configuration.
+
+* An `implementation of OptionContainer was added for iOS <https://github.com/beeware/toga/pull/2259>`__.
+
+* A number of improvements were made to image handling, including `optimizations when creating images from internal APIs <https://github.com/beeware/toga/pull/2263>`__, and a `correction to flexible image resizing behavior <https://github.com/beeware/toga/pull/2276>`__.
+
+* A `long standing bug that would cause macOS tables to crash <https://github.com/beeware/toga/pull/2291>`__ was resolved.
+
+* The `ability to define platform-specific icons <https://github.com/beeware/toga/pull/2260>`__ was added to Toga.
+
+* `Toga 0.4.1 was released <https://pypi.org/project/toga/0.4.1/>`__, incorporating the features and bug fixes from the last 2 months.
```

Comment:
Might the GitHub release be a better hyperlink candidate for this?

https://github.com/beeware/toga/releases/tag/v0.4.1

Reply:
I consider the PyPI release to be canonical, as that's the source of the release (including the fact that the artefact can't be modified). Honestly, I'd rather there wasn't a Github release page, but it's required for the release process, so I can live with it.